src/osm_configurator/view/freezable.py

```diff
@@ -13,6 +12,6 @@ def freeze(self):
     @abstractmethod
     def unfreeze(self):
         """
-        If this method is called, the frame returns into its previous interactable state.
+        If this method is called, the frame returns into its previous intractable state.
```

Comment:
intractable is the wrong word. intercatable is right, maybe if it is seen as an typo maybe change it to interact able or something, but intractable is defentely wrong!

Reply:
ok